### ISSUE TITLE
CrewCab issue

### ISSUE THREAD

**Issue by nourelhoda89** [anthonykulis/automobile#91]
@anthonykulis 
Can you please make CrewCab in package edu.jalc.automobile.onlinebuilder.components.body.truck;
 accept Cabin instead of luxuryCabin  (line 12 and 13), I tried several other cab but all take certain types of cabins and seats which would not work for us.

**Comment by anthonykulis** [comment]
No. Just go ahead and make luxury cabins for crew cabs. They technically
are anyways.

**Comment by nourelhoda89** [comment]
Sorry I just read your reply!
For a starter luxuryCabin only take leatherSeat, and StandardCabin only take ClothSeat.
so if I use StandardCab which only take Standard Cabin I am limited to ClothSeat,
if I us ExtendedCab  which only take luxuryCabin i am limited to LeatherSeat
and CrewCab as it is will limit seat to Leather as well. 
for choosing a Seat to work in the builder I had to make all of them extends TruckSeat, which extends Seat, not any specific types of seat. So we are stuck to either change CrewCab  to take all types of Cabin, not just Luxury,  or use StandardCab but change StandardCabin to accept Seat not just ClothSeats.
 Either one  should not hurt any ones code since its giving more options not limiting them. 
What do you think??
